.changeset/wicked-dancers-exercise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/inline-code': feat
+'@twilio-paste/core': feat
```

Comment:
are `feat` and `minor` interchangeable? 👀

Reply:
:doh: theyre not. I changed it from patch incorrectly